themes/src/main/resources/theme/keycloak-preview/account/messages/messages_en.properties

```diff
@@ -68,3 +68,22 @@ setUp=Set Up
 passwordless=Passwordless
 lastUpdate=Last Update
 unknown=Unknown
+
+# Applications page
+applicationsPageTitle=Applications
+internalApp=Internal
+thirdPartyApp=Third-party
+offlineAccess=Offline Access
+inUse=In use
+notInUse=Not in use
+applicationDetails=Application Details
+client=Client
+description=Description
+baseUrl=URL
+accessGrantedOn=Access granted on
+removeButton=Remove
```

Comment:
We should change this to "Remove Access" to be compliant with the `infoMessage`.